test/kata7.getEmployerRole.test.js

```diff
@@ -1,5 +1,29 @@
 const { getEmployerRole } = require("../src");
 
 describe("getEmployerRole", () => {
-  test("returns the employee's role in the company", () => {});
-});
+  test("returns the employee's role in the company", () => {
```

Comment:
The formatting is just out here, the logic is fine. Would be really good to get your IDE set up to automatically format your work on save, as it would make things easier to read and save you some extra typing to boot. In VSCode, if you go into your settings and search for 'format on save', turn this on (in both your workspace settings AND user settings), when you save your file it will automatically format it for you so you don't even have to remember to do it!

Rather than comment similar throughout your work here, Ill just point out logic or other non-formatting type issues as if you were to do this with your project, it will fix all the extra whitespace/not enough spacing/unnecessary syntax for you!